### PR TITLE
Add build to Travis to catch more errors.

### PR DESCRIPTION
**Pull request by Tj-Tracy**
My only concern with this is the Gatsby cache breaking. I'm not sure what travis caches, or how they would interact, but I suppose if it becomes a problem we could also add a `clean` step.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,7 +17,7 @@ down: ## stop and remove learner-portal container
 	docker-compose down
 
 npm-install-%: ## install specified % npm package on the learner-portal container
-	docker exec npm install $* --save-dev
+	docker exec frontend-app-learner-portal_web_1 npm install $* --save-dev
```
Comment (georgebabey) [inline Makefile]: Should probably give the container_name in the docker-compose.yml rather than assume the name here. Also seems unrelated to the above change so maybe just push a separate PR for the Makefile/docker-compose changes

Comment (Tj-Tracy) [reply Makefile]: Whoops, I forgot I did this. It's actually also in @tuchfarber's pr. I'm going to remove it.

Comment (tuchfarber) [reply Makefile]: oops. That wasn't supposed to make it into my PR. That was supposed to be a temporary quick fix.
